config/GALE01/symbols.txt

```diff
@@ -8593,7 +8593,7 @@ gm_8016AEDC = .text:0x8016AEDC; // type:function size:0x10 scope:global
 gm_8016AEEC = .text:0x8016AEEC; // type:function size:0x10 scope:global
 gm_8016AEFC = .text:0x8016AEFC; // type:function size:0x10 scope:global
 gm_8016AF0C = .text:0x8016AF0C; // type:function size:0x7C scope:global
-gm_8016AF88 = .text:0x8016AF88; // type:function size:0x7C scope:global
+GetMatchTimer = .text:0x8016AF88; // type:function size:0x7C scope:global
```

Comment:
should rename the corresponding function in gm_16AE, otherwise the decomp.dev bot thinks this function has regressed